README.md

```diff
@@ -166,12 +174,14 @@ A batch file comprises three different line types which exist on a one : many :
 | Water body Buffering | 80008 |
 | Pilot Participation Payment | 80009 |
 
-## Running the application
+# Setup
 
 The application is designed to run in containerised environments, using Docker Compose in development and Kubernetes in production.
 
 - A Helm chart is provided for production deployments to Kubernetes.
```

Comment:
I'd tweak this wording to not say production as we deploy to Kubernetes in every environment.